cypress/support/step_definitions/accountDetails/businessUnitAndDefendantType.ts

```diff
@@ -56,8 +56,8 @@ When('I select parent or guardian to pay', () => {
 When('I select company', () => {
   cy.get('input[id="company"]').check().should('be.checked');
 });
-Then('I click on cancel', () => {
-  cy.contains('Cancel').click();
+Then('I click on {string}', (cancelLink: string) => {
+  cy.contains('a', cancelLink).click();
```

Comment:
there is already a step for this in genericSteps/linkSteps.ts

Reply:
There was already a step for cancel however it was not 'a' tag. In this ticket the selector was changed

Reply:
In the genericSteps/linkSteps.ts file there are a couple existing link steps.
<img width="518" alt="image" src="https://github.com/user-attachments/assets/9ee23e5f-4b78-49d4-97ab-18b3b43cd6b4">

Reply:
Using already developed step now

Reply:
great, can this step please be removed as it is a duplicate of the existing step?

Reply:
This step can be removed as the existing step is being used.

Reply:
the step using, deleted that I've created